### PR TITLE
LastHash

### PR DESCRIPTION
**Pull request by phlllpe**
Method to get last commit hash in git.
The preg_replace function is removing "commit string" and get first eight characters

### CODE REVIEW

File: src/Git.php [hunk 1]
```diff
@@ -117,4 +117,16 @@ protected function execute($command)
...
+    /**
+     * @return string
+     */
+    public function getLastHash()
+    {
+        $result = $this->execute("git log -1 --stat");
+        if ($result) {
+            return preg_replace('#^([\w+]{8}).+#', '\\1', preg_replace('#^[^\s]+\s#', '', current($result)));
+        }
+        return null;
```
Comment (GrahamCampbell) [inline src/Git.php]: not needed

Comment (GrahamCampbell) [inline src/Git.php]: just ommit the return statament

File: src/Git.php [hunk 2]
```diff
@@ -117,4 +117,16 @@ protected function execute($command)
 
         return $output;
     }
+    
+    /**
+     * @return string
```
Comment (GrahamCampbell) [inline src/Git.php]: string|null

**Comment by dudemelo** [comment]
Why not use ```git rev-parse HEAD``` or ```git log -1 --format="%H"``` that returns the commit hash?

**Comment by sebastianbergmann** [comment]
Closing because I rather not add additional functionality to this component.

**Pull request closed without merging**